libs/dextool/source/dextool/compilation_db/package.d

```diff
@@ -1180,3 +1216,29 @@ unittest {
 
     found[0].absoluteFile.baseName.shouldEqual("file3.cpp");
 }
+
+@("shall extract filepath from includes correctly when there is spaces in the path")
+unittest {
+    auto cmd = toCompileCommand("/home", "file.cpp", [
+            "-I", "dir with spaces/src"
+            ], AbsoluteCompileDbDirectory("/home"), null);
+    auto pargs = cmd.get.parseFlag(defaultCompilerFilter, Compiler.init);
+    pargs.cflags.shouldEqual(["-I", "/home/dir with spaces/src"]);
+    pargs.includes.shouldEqual(["/home/dir with spaces/src"]);
+}
+
+@("shall handle path with spaces, both as separate string and combined")
+unittest {
+    auto cmd = toCompileCommand("/project", "file.cpp", [
+            "-I", "separate dir/with space", "-Icombined dir/with space/src"
```

Comment:
See my previous comment for the first argument.
The second one should probably be:
```d
`-I"combined dir/with space/src"`
```